test/e2e/spec/e2e_spec.rb

```diff
@@ -58,6 +58,48 @@
     end
   end
 
+  describe "E2E Balance -> Sign -> Submit" do
+    plutus_dir = "../../lib/shelley/test/data/plutus/"
+    plutus_scripts_ok = ["token-account-2.json",
+                         "multisig-success-2.json",
+                         "uniswap-7.json",
+                         "crowdfunding-success-4.json",
+                         "pubkey-2.json",
+                         "future-settle-early-2.json",
+                         "future-increase-margin-2.json",
+                         "escrow-refund-2.json",
+                         "future-pay-out-2.json",
+                         "uniswap-2.json",
+                         "ping-pong_2-2.json",
+                         "multisig-failure-2.json",
+                         "currency-2.json"]
+    plutus_scripts_ok.each do |script|
+      it "#{script}" do
+        testnet_address = SHELLEY.addresses.list(@wid)[0]['id']
+        payload = JSON.parse(File.read(plutus_dir + script))
+        # change all addresses in example payloads to testnet addresses
+        payload['inputs'].map {|i| i['txOut']['address'] = testnet_address}
+
+        tx_balanced = SHELLEY.transactions.balance(@wid, payload)
+        expect(tx_balanced).to be_correct_and_respond 202
+
+        tx_signed = SHELLEY.transactions.sign(@wid, PASS, tx_balanced['transaction'])
+        expect(tx_signed).to be_correct_and_respond 202
+
+        tx_submitted = PROXY.submit_external_transaction(Base64.decode64(tx_signed['transaction']))
+        expect(tx_submitted).to be_correct_and_respond 500
+        expect(tx_submitted.to_s).to include "created_invalid_transaction"
```

Comment:
trying to submit some examples (the are that are successfully balanced, others aren't), but they're all rejected by the node.